.env.test

```diff
@@ -1,3 +1,2 @@
-# # Authorization should not be necessary for testing
+NODE_PATH=src/core_modules
```

Comment:
What is this used for?  It is also duplicated in the `test` script in `package.json`

Reply:
To add on this I think we might not need the whole file. But I could be wrong about that.